src/tokens.md

```diff
@@ -611,3 +611,8 @@ them are referred to as "token trees" in [macros].  The three types of brackets
 [extern]: items/external-blocks.html
 [struct expressions]: expressions/struct-expr.html
 [tuple index]: expressions/tuple-expr.html#tuple-indexing-expressions
+[Wildcard patterns]: patterns.html#wildcard-pattern
```

Comment:
Can you alphabetize these? You don't need to alphabetize all of the ones that are already here, but at least from `extern` downwards (since the three already at the end are alphabetized by coincidence). Uppercase sorts before all lowercase.